### ISSUE TITLE
Invalid request format: json: cannot unmarshal bool into Go struct field BidRequest.test of type int8

### ISSUE THREAD

**Issue by nanek** [prebid/prebid-mobile-ios#61]
We had prebid mobile integrated and working. However of late, I'm seeing this error message. I'm using v0.2.0 with mopub and appnexus. Is this expected?

```
Prebid: Bid response from Prebid Server: Invalid request format: json: cannot unmarshal bool into Go struct field BidRequest.test of type int8
Prebid: Error parsing ad server response
````

**Comment by ppuviarasu** [comment]
Are you using the cocoapods version or the code downloaded from github?

**Comment by nanek** [comment]
Cocoapods.

**Comment by ppuviarasu** [comment]
Yes there is a bug in that version. Will be releasing a new version today... Kindly bear with us... The Github code is latest updated

**Comment by ppuviarasu** [comment]
We have released a new version of the SDK today 0.2.1... Please take the latest version & integrate
Thanks
